src/main/java/com/uber/cadence/samples/hello/HelloActivityRetry.java

```diff
@@ -108,7 +110,7 @@ public static void main(String[] args) {
     // ClientOptions.newBuilder().setRpcTimeout(5 * 1000).build();
     WorkflowClient workflowClient =
         WorkflowClient.newInstance(
-            new WorkflowServiceTChannel(ClientOptions.defaultInstance()),
+            new Thrift2ProtoAdapter(IGrpcServiceStubs.newInstance()),
```

Comment:
Wondering how to test if all of the samples work correctly after this change? Assuming just run and execute manually?

Reply:
tested manually